test/Microsoft.Health.Extensions.Fhir.R4.UnitTests/FhirServiceExceptionProcessorTests.cs

```diff
@@ -19,14 +18,14 @@ namespace Microsoft.Health.Extensions.Fhir.R4.UnitTests
 {
     public class FhirServiceExceptionProcessorTests
     {
-        private static readonly Exception _fhirForbiddenEx = new FhirOperationException("test", HttpStatusCode.Forbidden);
-        private static readonly Exception _fhirNotFoundEx = new FhirOperationException("test", HttpStatusCode.NotFound);
-        private static readonly Exception _fhirBadRequestEx = new FhirOperationException("test", HttpStatusCode.BadRequest);
+        private static readonly Exception _httpRequestForbiddenEx = new HttpRequestException("test", inner: null, HttpStatusCode.Forbidden);
+        private static readonly Exception _httpRequestNotFoundEx = new HttpRequestException("test", inner: null, HttpStatusCode.NotFound);
+        private static readonly Exception _httpRequestBadRequestEx = new HttpRequestException("test", inner: null, HttpStatusCode.BadRequest);
         private static readonly Exception _argEndpointNullEx = new ArgumentNullException("endpoint");
         private static readonly Exception _argEndpointEx = new ArgumentException("endpoint", "Endpoint must be absolute");
         private static readonly Exception _argEx = new ArgumentException("test_message", "test_param");
         private static readonly Exception _uriEx = new UriFormatException();
-        private static readonly Exception _httpNotKnownEx = new HttpRequestException("Name or service not known");
+        private static readonly Exception _httpNotKnownEx = new HttpRequestException("Name or service not known", inner: null, HttpStatusCode.NotFound);
```

Comment:
Please refer to my previous comment about the `HttpRequestException` with the `Message` of "Name or service not known" not having a `StatusCode`. Not sure if this is still true with the new FhirClient.

Reply:
Pushed new changes to FhirServiceExceptionProcessorTests. Please refer to comment above